flatpak-builder/index.js

```diff
@@ -113,6 +113,16 @@ const modifyManifest = (manifest, runTests = false) => {
   return manifest
 }
 
+/**
+ * Gets the modified manifest as a YAML or JSON file
+ *
+ * @param {PathLike} manifestPath Where to save the flatpak manifest
+ * @returns {PathLike} the manifest
+ */
+const getModifiedManifestPath = async (manifestPath) => {
```

Comment:
```suggestion
const getModifiedManifestPath =  (manifestPath) => {
```

doesn't need to be async

Reply:
Yup, hold on, I'm testing it in my repo and there are more changes in the queue!

Reply:
Changed that